addons/website_sale_slides/tests/test_course_purchase_flow.py

```diff
@@ -61,6 +61,7 @@ def test_course_purchase_flow(self):
         # Step 4: Same test as salesman
         salesman_sale_order = self.env['sale.order'].with_user(self.user_salesman).create({
             'partner_id': self.user_portal.partner_id.id,
+            'user_id': self.user_salesman.id,
```

Comment:
This should not be necessary, as there is a with_user used for the create. Why this change ?